csrc/scheduler/hopper_multi_matmul.cpp

```diff
@@ -539,23 +538,41 @@ void HopperMultipleMatmulScheduler::scheduleEpilogue() {
         transformLikeMmaOutput(tv, /*is_mma_result=*/false);
       }
 
-      auto s = mma_utils::MmaSwizzler::scheduleMmaOutputAllocation(
-          dc->getLoopDomain());
-      dc->setLoopDomain(s.as<IterDomain*>());
-      dc->setAllocationDomain(s.as<IterDomain*>(), true);
-
-      scheduler_utils::BoundedDirectionalTransformPropagator::backward(
-          dc,
-          -1,
-          propagate_to,
-          scheduler_utils::BoundedDirectionalTransformPropagator::Options()
-              .propagateParallelType());
+      // Should not propagate if the dc is a mma output as the mma output has
+      // already been scheduled.
+      if (std::find(mma_results_.begin(), mma_results_.end(), dc) ==
```

Comment:
I wonder if a lambda is more readable for this `std::find`. We used it twice.
```cpp
      auto is_not_mma_result = [](TensorView* tv) { std::find(mma_results_.begin(), mma_results_.end(), tv) == mma_results_.end() };
      if (is_not_mma_result(dc)) {
          // schedule(dc);
      }
```

Reply:
I went with a bool instead of a lambda but agreed about readability.